### PR TITLE
AV-156960 Add vip placement from vip network for nsxt vlan (cherry-pick)

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

**Comment by akshayhavile** [comment]
@aaha97 We need to run tls-ingress, lbsvc and tls-route test suite against the changes.

**Review by akshayhavile — COMMENTED** [review]
@aaha97 Overall changes looks good. But as an extension we can use ipv6 address also in placement network. CC: @apalsule

File: internal/rest/avi_obj_vsvip.go [hunk 1]
```diff
@@ -128,6 +129,26 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 					if vsvip_meta.VipNetworks[0].V6Cidr != "" {
 						lib.UpdateV6(vip, &vsvip_meta.VipNetworks[0])
 					}
+					if lib.GetCloudType() == lib.CLOUD_NSXT &&
```
Comment (apalsule) [inline internal/rest/avi_obj_vsvip.go:132]: Seems like same code duplicated at 2 places, can you make a function or something out of it

Comment (aaha97) [reply internal/rest/avi_obj_vsvip.go:132]: removed duplication

**Review by akshayhavile — APPROVED** [review]

**Comment by aaha97** [comment]
chery-picked to master as a different PR #1003

**Pull request closed without merging**